conwayconsole/Program.cs

```diff
@@ -93,6 +93,8 @@ public static async Task Main(params string[] args)
             DateTime now = DateTime.UtcNow;
             TimeSpan elapsed = now.Subtract(lastLoopTime);
             TimeSpan delay = TimeSpan.FromMilliseconds(options.Delay).Subtract(elapsed);
+            
+            Console.WriteLine($"\u001b[0;0H({board.Width}x{board.Height}) ({window.Width}x{window.Height}) ({window.X}, {window.Y})");// \u001b[0;0 is the special code to send the line to the top of the window
```

Comment:
The special characters here are a constant in BoardConsoleExtensions. It is currently private, but if you change it to public, you can (and should) use it here, so that this piece of "magic" is only in one place in our project